.php_cs.dist

```diff
@@ -15,7 +15,6 @@ $config
         'array_syntax' => array('syntax' => 'long'),
         'binary_operator_spaces' => false,
         'concat_space' => array('spacing' => 'one'),
-        'no_unused_imports' => false,
```

Comment:
Since `no_unused_imports` is enabled as part of the `@Symfony` rule set, we implicitly enable it by removing this rule here.